about.html

```diff
@@ -10,7 +10,10 @@
 <body>
 
     <div id="banner-container"></div>
-
+    <div id="box1">
+        <h2 class="heading">What We Do</h2>
+        <p class="mission">We seek to provide computing solutions for non-profits and local businesses in Madison, Wisconsin </p>
```

Comment:
I personally have one suggestion for this PR:

```suggestion
        <p class="mission">We seek to provide computing solutions for non-profits and local businesses in Madison, Wisconsin.</p>
```

If I recall correctly, there is a button called "Commit Suggestion" (more info [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes)) that you can click on to directly apply this hulk to your branch ([guechhouth:about](https://github.com/guechhouth/CodeForGoodWebsite/tree/about)). If you deem this to be is a desirable change, consider approving this suggestion by clicking on that button; I recommend using the commit message: "Add missing end punctuation".